drivers/hmis/app/graphql/concerns/graphql_application_helper.rb

```diff
@@ -14,6 +14,10 @@ def hmis_user
     Hmis::Hud::User.from_user(current_user)
   end
 
+  def true_hmis_user
+    Hmis::Hud::User.from_user(true_user)
+  end
```

Comment:
maybe check if impersonating to avoid an extra query

```suggestion
  def impersonating?
    current_user != true_user
  end
  
  def true_hmis_user
    impersonating? ? Hmis::Hud::User.from_user(true_user) : hmis_user
  end
```